packages/ui/Transition/Transition.js

```diff
@@ -523,13 +519,13 @@ class Transition extends Component {
 	};
 
 	handleTransitionEnd = (ev) => {
-		forwardTransitionEnd(ev, this.props);
+		forwardCustom('onTransitionEnd')(ev, this.props);
```

Comment:
What about replacing `ev` with `null` to avoid any misunderstanding later? :)

Reply:
🆗